pycqed/analysis/fitting_models.py

```diff
@@ -310,33 +312,10 @@ def HangerFuncAmplitude(f, f0, Q, Qe, A, theta):
     return abs(A * (1. - Q / Qe * np.exp(1.j * theta) / (1. + 2.j * Q * (f / 1.e9 - f0) / f0)))
 
 
-def HangerFuncComplex(f, pars):
```

Comment:
I'm assuming this is being deleted on purpose?

Reply:
The standard complex hanger function should, just like the standard for the fitting models, take floats as arguments.
In order no to loose functionality the function taking a list of parameters as been renamed to hanger_func_complex_SI_pars.
I've added a comment to this.